Analysis/DEEP2_R23_O32/Plotting/te_metal_plots.py

```diff
@@ -204,3 +211,412 @@ def plotting_te_metal(fitspath, out_pdf):
     pdf_pages.close()
         
 
+
+
+def Jiang_comparison():
+    #log(R23) = a +bx+cx^2 - d(e+x)y
+    #x = 12+log(O/H)
+    #y = log(O32)
+    
+
+    
+    fitspath = '/Users/reagenleimbach/Desktop/Zcalbase_gal/R23O32_Manual_0417/'
+    validation = asc.read(fitspath + 'bin_validation.revised.tbl')
+    temp_tab = asc.read(fitspath + 'bin_derived_properties.tbl')
+
+
+    pdf_pages = PdfPages(fitspath+'comparsion_Jiang_Zcal.pdf')
+
+    
+    bin_ID = temp_tab['bin_ID']
+    lR23_comp = temp_tab['logR23_Composite']
+    lO32_comp = temp_tab['logO32_Composite']
+    zmetal = temp_tab['12+log(O/H)']
+
+    valid = validation['Detection']
+    detect = np.where((valid ==1.0))[0]
+    rlimit = np.where((valid ==0.5))[0]
+
+    valid_ID = bin_ID[detect]
+
+    lR23 = lR23_comp[detect]
+    lO32 = lO32_comp[detect]
+
+    rlR23 = lR23_comp[rlimit]
+    rlO32 = lO32_comp[rlimit]
+
+
+    metal_det = zmetal[detect]
+    metal_rl = zmetal[rlimit]
+
+
+    derived = asc.read(fitspath_ini +'DEEP2_R23_O32_derived.tbl')
+    derived_MACT = asc.read(fitspath_ini +'MACT_R23_O32_derived.tbl')
+    
+    #DEEP2 Derived 
+    er_R23 = derived['R23'].data
+    er_O32 = derived['O32'].data
```

Comment:
I originally had it as the "der" for derived, but then I realized that I needed to take the log of the value so I took to d off because I didn't want to go through and change it in the graphing.